package.json

```diff
@@ -511,6 +655,31 @@
             "type": "string"
           }
         },
+        "r.rMarkdownCodeLens.option": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "default": [
+            "r.runCurrentChunk",
+            "r.runAboveChunks",
+            "r.runCurrentAndBelowChunks",
+            "r.runAllChunks",
+            "r.goToPreviousChunk",
+            "r.goToNextChunk"
+          ],
+          "description": "Available options:\n\nr.selectCurrentChunk\nr.runCurrentChunk\nr.runAboveChunks\nr.runCurrentAndBelowChunks\nr.runBelowChunks\nr.runAllChunks\nr.runPreviousChunk\nr.runNextChunk\nr.goToBeginningOfChunk\nr.goToEndOfChunk\nr.goToPreviousChunk\nr.goToNextChunk\nr.goToFirstChunk\nr.goToLastChunk\n\n----------------------------------\nCustomize both the options AND its orders (that is, code lens respect user-specified orders):"
+        },
+        "r.rMarkdownCodeLens.enableCodeLens": {
+          "type": "boolean",
+          "default": true,
+          "description": "Enable RMarkdown code lens"
+        },
+        "r.rMarkdownCodeLens.chunkBackgroundColor": {
+          "type": "string",
+          "default": "rgba(128, 128, 128, 0.1)",
+          "description": "RMarkdown code lens chunk background color in RGBA or RGB value. Defaults to rgba(128, 128, 128, 0.1). Reload VS Code after changing settings. Only works when option `rMarkdownCodeLens.enableCodeLens` is set to true (default).\n\nLearn how to set colors: https://www.w3schools.com/css/css_colors_rgb.asp.\n\nExamples for syntax rgba(<red>, <green>, <blue>, <alpha>):\nrgba(128, 128, 128, 0.1)\nrgba(128, 128, 128, 0.3)\nrgba(255, 165, 0, 0.1)\n\n"
```

Comment:
See https://github.com/Ikuyadeu/vscode-R/pull/465/files#r531823145. CodeLens and background decoration are separate things and are independent of each other.

Reply:
Made the customization of CodeLens and chunk background color independent